src/Umbraco.Web.UI.Client/package.json

```diff
@@ -37,6 +37,7 @@
     "jquery-ui-touch-punch": "0.2.3",
     "lazyload-js": "1.0.0",
     "moment": "2.22.2",
+    "monaco-editor": "^0.19.3",

```

Comment:
Perhaps it should just lock to this version? :-)

Reply:
If this POC ever gets considered/accepted to be put in then, yes that's a good idea to lock the version